src/components/SuccessContent.tsx

```diff
@@ -35,13 +51,35 @@ const SuccessContent = ({ title, email }) => {
       <Typography variant="body1">
         {t(AUTH.SIGN_IN_SUCCESS_EMAIL_PROBLEM)}
       </Typography>
+      <br />
+      <Button
+        variant="outlined"
+        color="primary"
+        onClick={onClickResendEmail}
+        id={RESEND_EMAIL_BUTTON_ID}
+        disabled={clicked}
+      >
+        {t(AUTH.RESEND_EMAIL_BUTTON)}
+      </Button>
```

Comment:
I think there's too much duplication in this PR. I would personally assume this `Resend Email` button should get the email and sign in with it, independently from sign in and sign up screens.